test/common/tcp_proxy/upstream_test.cc

```diff
@@ -56,13 +57,10 @@ template <typename T> class HttpUpstreamTest : public testing::Test {
   NiceMock<Server::Configuration::MockFactoryContext> context_;
 };
 
-using testing::Types;
+INSTANTIATE_TEST_SUITE_P(H1H2Codecs, HttpUpstreamTest,
+                         ::testing::Values(Http::CodecType::HTTP1, Http::CodecType::HTTP2));
```

Comment:
What's the reason HTTP3 isn't included here and below? e.g. if H3 was using the H2 upstream impl before and should still work?

Reply:
good catch, thanks!!